src/index.ts

```diff
@@ -1,16 +1,49 @@
+import packageInfo from '../package.json'
```

Comment:
The whole `package.json` content has got into the distributive files because of this. Not a problem actually, just an excess code in the distributive files. Though it's removed by Terser.

Reply:
I will fix it in a separate pull request.